static/js/chart/draw.ts

```diff
@@ -104,7 +107,10 @@ function addXAxis(
   const axis = svg
     .append("g")
     .attr("class", "x axis")
-    .attr("transform", `translate(0, ${height - MARGIN.bottom})`)
+    .attr(
+      "transform",
+      `translate(${YLABEL.rightMargin}, ${height - MARGIN.bottom})`
```

Comment:
We should only apply right margin if we are about to add a label to the margin. This would currently affect the charts on the place pages, which shouldn't change.

Reply:
changed it back